packages/monaco/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@theia/core": "^0.5.0",
     "@theia/editor": "^0.5.0",
+    "@theia/editor-preview": "^0.5.0",
```

Comment:
it will force editor preview on everybody, so far it was not used much because of known issues